lib/active_model/serializer/field.rb

```diff
@@ -0,0 +1,56 @@
+module ActiveModel
+  class Serializer
+    # Holds all the meta-data about a field (i.e. attribute or association) as it was
```

Comment:
grammar: `s/i.e./e.g.`

reason: meaning is to change 'that is' to 'for example'